.travis.yml

```diff
@@ -10,16 +10,20 @@ script:
   - jekyll build --source docs
   - htmlproofer ./_site --disable-external --url-swap "\/inkscape-bobbinlace\/:/" --assume-extension
 before_deploy:
-  - zip -r -q inkscape-bobbinlace-plugins.zip *.inx *.py
+  - zip -q lace_circular_ground.zip lace_circular_ground.*
```

Comment:
Not related with this pull request but I must be blindfolded somehow. Line 11 above works for all  links in the sidebar but one. 
 
https://github.com/d-bl/inkscape-bobbinlace/blob/dc14be019c3e5817b86f17cc0484ee6f9cc38f2b/.travis.yml#L11

https://github.com/d-bl/inkscape-bobbinlace/blob/2ad4383f23164bcccc3763cd54e0269e5fb9aa91/docs/_includes/Sidebar.html#L12-L17